### PR TITLE
Add tests for `sf::Text`

### PR DESCRIPTION
**Pull request by ChrisThrasher**
## Description

Not sure why I didn't write these tests earlier but anyways, here they are! While I was at it I found a nice simplification to make to `sf::Text` itself. We were writing a lot of default special member functions that we didn't need to write.

### CODE REVIEW

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Review by binary1248 — APPROVED** [review]

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

File: test/Graphics/Text.test.cpp [hunk 1]
```diff
@@ -1,9 +1,142 @@
...
+        STATIC_CHECK(std::is_copy_assignable_v<sf::Text>);
+        STATIC_CHECK(std::is_nothrow_move_constructible_v<sf::Text>);
+        STATIC_CHECK(std::is_nothrow_move_assignable_v<sf::Text>);
+    }
+
+    const sf::Font font;
+
+    SECTION("Construction")
+    {
+
```
Comment (kimci86) [inline test/Graphics/Text.test.cpp:26]: empty line

**Review by kimci86 — APPROVED** [review]

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Comment by ChrisThrasher** [comment]
Screenshot 2023-08-05 at 6 35 18 PM

🥳

**Pull request merged**